triton/kubernetes/FIL/interact_with_triton.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import tritonclient.http as triton_http\n",
+    "import tritonclient.grpc as triton_grpc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "port_cmd = \"kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name==\\\"http2\\\")].port}'\"\n",
+    "host_cmd = \"kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'\"\n",
+    "\n",
+    "port = subprocess.check_output(port_cmd.split()).decode('utf-8').replace(\"'\", \"\")\n",
+    "host = subprocess.check_output(host_cmd.split()).decode('utf-8').replace(\"'\", \"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Set up both HTTP and GRPC clients. Note that the GRPC client is generally\n",
```

Comment:
Just wanted to note that with my Helm chart, due to the way I set up Istio gateway and virtual service, the Triton deployment can only accept HTTP or GRPC format depends on the config of helm chart.

Reply:
Yes, thanks for the reminder. Was planning to update this so either could be used.

Reply:
Updated to support both in the same deployment.

Reply:
Thank you, Devin! I see you used the default TCP port for GRPC, could you please let me if TCP port/protocol is working in terms of loading balance GRPC traffic? For example, if you have 1 GRPC client and send all the requests.